code/game/atoms_movable.dm

```diff
@@ -555,9 +560,39 @@
 	SHOULD_NOT_OVERRIDE(TRUE)
 	CRASH("Uncross() should not be being called, please read the doc-comment for it for why.")
 
+/**
```

Comment:
You still copied and pasted this--Uncross is right above this.

Reply:
oh wtf i did

Reply:
gone